app/data/flags.json

```diff
@@ -115,5 +115,9 @@
   "SILLY_CLOWNS": {
     "label": "Silly clowns",
     "defaultValue": false
+  },
+  "ANALYTICS": {
+    "label": "Street Analytics",
```

Comment:
Cool, let's move this up to where the "GEOTAG" flag is because they occupy conceptually similar parts of the UI and rename the label to "UI – Street analytics". (The lower case "a" is because we have guidelines for [how we label things in the UI](https://streetmix.readthedocs.io/en/latest/contributing/content-styleguide/?highlight=labels#sentence-case), and this would be good to keep in mind in all future implementations of things.)